### PR TITLE
Add CONTRIBUTING file and section to docs

### PR DESCRIPTION
**Pull request by ikalnitsky**
[Review on Reviewable](https://reviewable.io/reviews/ikalnitsky/holocron/180)

### CODE REVIEW

File: CONTRIBUTING [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+enough to point the following things:
+
+* Steps to reproduce.
+* Expected behaviour.
+* Actual behaviour.
+* Holocron version.
+* Python version.
+* Operating System.
+
+Since `Issue Tracker`_ is used not only to track bugs, please attach ``bug``
```
Comment (ikalnitsky) [inline CONTRIBUTING]: Maybe replace `please attach` with `please do not forget to attach` ?

File: CONTRIBUTING [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+together!
+
+
+Reporting Bugs
+==============
+
+.. warning::
+
+   Bug reports are hugely important! Before you raise one, though, please
+   check through the `Issue Tracker`_, both open and closed, to confirm that
```
Comment (PropheticBird) [inline CONTRIBUTING]: through can be removed

File: CONTRIBUTING [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+   the bug hasn’t been reported before. Duplicate bug reports are a huge
+   drain on the time of other contributors, and should be avoided as much
+   as possible.
+
+Each time you are about to report a bug please provide as much information
+as possible, but keep focusing on things that matters. Usually it'd be
+enough to point the following things:
```
Comment (PropheticBird) [inline CONTRIBUTING]: point -> specify

File: CONTRIBUTING (hunk 1 above)
Comment (PropheticBird) [inline CONTRIBUTING]: yes

File: CONTRIBUTING [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+==================
+
+There are two ways to submit a patch. The first one is by sending a pull
+request. It's the preferred way and is recommended for general usage. If
+by some reason you don't have a GitHub account, there's another option -
+to send a patch via email to project maintainers.
+
+Before you send a patch, please make sure that:
+
+* Your changes is covered by tests.
```
Comment (PropheticBird) [inline CONTRIBUTING]: are

Comment (PropheticBird) [inline CONTRIBUTING]: i don't this this should be a requirement, rather a nice to have thing

File: CONTRIBUTING [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+* ``tox`` run reports success.
+
+
+Developing Extensions
+=====================
+
+Holocron provides a flexible extension system for developers. If you're
+a frontend web developer and you have a beautiful web design, please
```
Comment (PropheticBird) [inline CONTRIBUTING]: web design *skills* ?

Comment (PropheticBird) [inline CONTRIBUTING]: i don't get this sentence

File: CONTRIBUTING (hunk 2 above)
Comment (PropheticBird) [inline CONTRIBUTING]: !fix

File: CONTRIBUTING (hunk 3 above)
Comment (PropheticBird) [inline CONTRIBUTING]: !fix

File: CONTRIBUTING (hunk 1 above)
Comment (PropheticBird) [inline CONTRIBUTING]: !resolved

Comment (PropheticBird) [inline CONTRIBUTING]: !fix

File: CONTRIBUTING (hunk 4 above)
Comment (PropheticBird) [inline CONTRIBUTING]: !fix

File: CONTRIBUTING (hunk 5 above)
Comment (PropheticBird) [inline CONTRIBUTING]: !resolve

File: CONTRIBUTING (hunk 2 above)
Comment (ikalnitsky) [inline CONTRIBUTING]: !fixed

File: CONTRIBUTING (hunk 3 above)
Comment (ikalnitsky) [inline CONTRIBUTING]: !fixed

File: CONTRIBUTING (hunk 1 above)
Comment (ikalnitsky) [inline CONTRIBUTING]: !fixed

File: CONTRIBUTING (hunk 4 above)
Comment (ikalnitsky) [inline CONTRIBUTING]: !fixed

File: CONTRIBUTING (hunk 5 above)
Comment (ikalnitsky) [inline CONTRIBUTING]: !fixed

**Pull request merged**